frontend/src/host_orchestrator/orchestrator/instancemanager.go

```diff
@@ -676,6 +676,8 @@ func fileExist(name string) (bool, error) {
 	}
 }
 
+const CVDCommandDefaultTimeout = 30 * time.Second
```

Comment:
30 seconds is too short for `cvd start`, make it at least 2 minutes.

Reply:
`cvd start` executions won't use this default, it will use 5 minutes, which I wanted to set to 1 minute in a another PR, but it's good to know 2 minutes it's a better number.